frontend/public/style/_common.scss

```diff
@@ -231,6 +231,11 @@ input[type=number]::-webkit-outer-spin-button {
   padding: 5px 0;
 }
 
+.co-toolbar__item--divider {
+  color: $color-pf-black-500;
```

Comment:
We should have some way of ensuring that this and `pairs-list__action-divider` use the same color (and maybe other styles). So either a common var for color or perhaps a general `co-action-divider` class that is used in both places.

Reply:
Now using a general `co-action-divider` class in both places.